src/core.js

```diff
@@ -36,7 +36,7 @@ QUnit.isLocal = !( defined.document && window.location.protocol !== "file:" );
 // Expose the current QUnit version
 QUnit.version = "@VERSION";
 
-function createModule( name, testEnvironment ) {
+function createModule( name, testEnvironment, skip ) {
```

Comment:
Initially, I tend to define all arguments as object and use destructuring. Something like this:
```js
function createModule( { name, testEnvironment, skip, todo } ) {
  // ...
}
```

This was my approach when I implemented all 3 methods in the first PR. But I also like your bellow suggestion to use `modifiers` as the third param